charts/generic-service/templates/ingress_dso.yaml

```diff
@@ -0,0 +1,40 @@
+{{- if and .Values.ingress.enabled .Values.ingress.dso_enabled -}}
+{{- $fullName := include "generic-service.fullname" . -}}
+{{- $svcPort := .Values.service.port -}}
+{{- $ingressPath  := .Values.ingress.path -}}
+{{- $hosts := or .Values.ingress.hosts (list .Values.ingress.host) }}
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: {{ $fullName }}
+  labels:
+    {{- include "generic-service.labels" . | nindent 4 }}
+  annotations:
```

Comment:
I've removed the unsupported mod security stuff as not applicable for dso hosting

Reply:
also removed the green annotation and weight